lmms_eval/tasks/step2_audio_paralinguistic/utils.py

```
@@ -185,85 +178,71 @@ def process_results(doc, result):
-            messages=[
-                {"role": "system", "content": "你是一个专业的文本评估助手"},
-                {"role": "user", "content": formatted_prompt}
-            ],
-            temperature=0
-        )
-        
+
+        response = client.chat.completions.create(model="gpt-3.5-turbo", messages=[{"role": "system", "content": "你是一个专业的文本评估助手"}, {"role": "user", "content": formatted_prompt}], temperature=0)
+
         result = response.choices[0].message.content.strip().lower()
         return 1 if result == "yes" else 0
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)